pdaltools/color.py

```diff
@@ -102,6 +103,44 @@ def download_image_from_geoplateforme(
         raise ValueError(f"Downloaded image is white, with stream: {layer}")
 
 
+def download_image(
+    proj, layer, minx, miny, maxx, maxy, pixel_per_meter, outfile, timeout, check_images
+):
+    size_x_p = (maxx - minx)
+    size_y_p = (maxy - miny)
+
+    # the image size is under SIZE_MAX_IMAGE_GPF
+    if size_x_p<SIZE_MAX_IMAGE_GPF and size_y_p<SIZE_MAX_IMAGE_GPF:
+        return download_image_from_geoplateforme(proj, layer, minx, miny, maxx, maxy, pixel_per_meter, outfile, timeout, check_images)
+
+    # the image is taller than the SIZE_MAX_IMAGE_GPF
+    # it's preferable to calcul it by paving
+    nb_cell_x = int(size_x_p / SIZE_MAX_IMAGE_GPF) + 1
+    nb_cell_y = int(size_y_p / SIZE_MAX_IMAGE_GPF) + 1
+
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        tmp_gpg_ortho = []
+        for l in range(0, nb_cell_y):
+            for c in range(0, nb_cell_x):
+
+                minx_cell = minx + c*SIZE_MAX_IMAGE_GPF
+                maxx_cell = minx_cell + SIZE_MAX_IMAGE_GPF
+                miny_cell = miny + l*SIZE_MAX_IMAGE_GPF
+                maxy_cell = miny_cell + SIZE_MAX_IMAGE_GPF
+
+                tmp_gpf_cell = tmp_dir_name + "/cell_"+str(c)+"_"+str(l)+".tif"
```

Comment:
utilise plutôt `os.path` ou `pathlib.Path` pour générer tes chemins, ça évitera d'avoir des problèmes de séparateurs entre windows et unix

proposition plus habituelle : 
```suggestion
                tmp_gpf_cell = Path(tmp_dir_name)  + f"cell_{c}_{l}.tif"
```